apps/comps/config/private.ts

```diff
@@ -78,6 +78,11 @@ const configSchema = z.strictObject({
   healthCheck: z.object({
     apiKey: z.string().optional(),
   }),
+  rewards: z.object({
+    slackWebhookUrl: z.string().default(""),
```

Comment:
This is not the complete rewards config used in `api` but it is sufficient for the rewards calculation.